roles/vdm/tasks/main.yaml

```diff
@@ -193,6 +193,14 @@
     - uninstall
     - update
 
+- name: Include Message broker
+  include_tasks: message_broker.yaml
+  when: PROVIDER == "azure"
```

Comment:
Should an additional check be added here to make sure `V4_CFG_MESSAGE_BROKER_HOST`, `V4_CFG_MESSAGE_BROKER_NAME`, & `V4_CFG_MESSAGE_BROKER_PASSWORD` is not null, otherwise I think the task in L13 in roles/vdm/tasks/message_broker.yaml would update the sitedefault.yaml with keys that have null values?

Reply:
Good point. I will update the task with additional conditions here: https://github.com/sassoftware/viya4-deployment/blob/IAC-1020/roles/vdm/tasks/message_broker.yaml#L25

Reply:
@jarpat Updated the code to add additional guards to add values to sitedefault only if not none. Also introduced another variable `V4_CFG_MESSAGE_BROKER_ENABLE` which will allow user to choose whether want the Cloud message broker like Azure service bus or continue with the default RabbitMQ.